### ISSUE TITLE
Logging in- does that take a while? 

### ISSUE THREAD

**Issue by walinchus** [themarshallproject/klaxon#335]
I set up the app and forked the repo and so forth. And Klaxon said it would send me an email to sign in. Does that process take a while? I tried a couple times and it said it would send an email but I checked Spam and everything but I still don't see anything. Thanks.

**Comment by GabeIsman** [comment]
Hey @walinchus,

Did you see the note about contacting SendGrid in the setup instructions? Unfortunately we've been having some trouble with new accounts having their mail held by them. That could be what's happening. You should receive any login email inside of 2-3 minutes if it's going to arrive.

Hope that helps!

**Comment by walinchus** [comment]
Hmm. I actually set up SendGrid a while ago. Let me try again. Thanks.

**Comment by walinchus** [comment]
You are awesome! That was it, as I hadn't used it in a while, SendGrid had deactivated me or something. Thanks.

**Comment by GabeIsman** [comment]
Great, glad that did the trick!
